scss/_patterns_lists.scss

```diff
@@ -154,6 +145,20 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
   }
 }
 
+%vf-list-item-dotted {
+  background-position-x: -1rem;
```

Comment:
What are these rem values based on? Can they be expressed as our standard spacing variables or do they need to be hardcoded as values?